capture/parsers/tcp.c

```diff
@@ -213,6 +216,9 @@ int tcp_packet_process(ArkimeSession_t *const session, ArkimePacket_t *const pac
             session->ackTime = (packet->ts.tv_sec - session->firstPacket.tv_sec) * 1000000 +
                                (packet->ts.tv_usec - session->firstPacket.tv_usec) + 1;
         }
+	// Send only the header up with a length 0, useful for tools like JA4Plus
+        if (pluginsCbs & ARKIME_PLUGIN_TCP)
+            arkime_plugins_cb_tcp(session, packet, 0, packet->direction);
```

Comment:
since there is no return here, why is this one needed?

Reply:
Corrected this, I moved this to where the return is when the length <= 0.